Roadmap.md

```diff
@@ -0,0 +1,81 @@
+
+🚧 [Implement a mono repo structure with `lerna`](https://github.com/aragon/aragon-cli/pull/325) 
+
+🚧 Finish the `aragon start` command [here](https://github.com/aragon/aragon-cli/pull/255/files) which is intended to be more development friendly that `aragon run`, i.e. [doesn't create a new dao every time](https://github.com/aragon/aragon-cli/issues/311), uses http rather than ipfs (to have hot/live reload for the frontends, and maybe something similar for the contracts, a watch → compile loop) 
+
+👾 [E2E tests with sharness](https://github.com/aragon/aragon-cli/issues/358)
+
+👾 Bugfixes, see all [here](https://github.com/aragon/aragon-cli/labels/bug)
+
+- [gasLimit](https://github.com/aragon/aragon-cli/issues/346) issue
+- [gas issue on dao acl grant](https://github.com/aragon/aragon-cli/issues/350)
+- [dao new fails with ganache 2.3](https://github.com/aragon/aragon-cli/issues/321) (low priority because pinning works alright at the moment)
+
+Note: We should write tests as we fix these to ensure no regressions!
+
+👾 Small features (low hanging fruit)
+
+- remove [hard-coded gas price](https://github.com/aragon/aragon-cli/issues/353) (would be nice to let the user decide how much)
+
+👾 [Claiming an aragonid](https://github.com/aragon/aragon-cli/issues/347) 
+
+👾 Document new features (dao token commands)
+
+👾 Restructure documentation on hack.aragon to include:
+
+- Quick start / Overview / Introduction
+- Guides around specific subjects (e.g.: Publish to different environments, Using a different Ethereum account, Propagating content, etc), similar to [this](https://github.com/aragon/aragon-react-boilerplate#publish). Maybe also link more in-depth guides such as [this](https://forum.aragon.org/t/guide-custom-aragon-organization-deployment-using-the-cli/507)
+- FAQ section. We have [this](https://hack.aragon.org/docs/cli-usage.html#troubleshooting-faq) currently, but it should be it's own section and much more elaborate (If there is a recurring question on #dev-help we will add it there)
+
+## Long-term
+
+- Types (use Flow or Typescript) because it will allow us to catch more "errors" at compile time, having commands silently fail without noticing (like [this](https://github.com/aragon/aragon-cli/pull/334#discussion_r248659171)) less often. Using Typescript is also great for intellisense support (having auto-completion from IDEs) and refactoring
+- Improve smart contract tooling using new Ethereum dev tools from 0x:
+    - `[sol-trace](https://sol-trace.com)`: Human-readable stack traces
+    - `[sol-coverage](https://sol-coverage.com)`*:* Solidity code coverage
+    - `[sol-profiler](https://sol-profiler.com)`: Gas profiling for Solidity
+    - `[sol-compiler](https://sol-compiler.com)`: Solidity compilation
+
+    We plan to prioritize `sol-trace` and to look into the benefits that the others tools could have on the developer experience and decide whether to work on that *short-term*.
```

Comment:
**Bret:** I'd wait until https://github.com/0xProject/0x-monorepo/issues/1520 is fixed, otherwise asking devs to spin up a geth node to do this is a bit cumbersome (i.e. I wouldn't bother until I had a failing test that was difficult debug by itself)